intera_interface/src/intera_interface/camera.py

```diff
@@ -110,6 +110,9 @@ def set_callback(self, camera_name, callback, callback_args=None,
         @param callback_args: additional arguments to pass to the callback
         @type queue_size: int
         @param queue_size: maximum number of messages to receive at a time
+        @type buff_size: int
+        @param buff_size: incoming message buffer size in bytes.
+            Defaults to handle head image of 4.1 MB.
```

Comment:
Right, but I'm looking for the documentation to specify the mapping between the 2^24 --> 4.1 MB. Perhaps we can have an additional line of explanation in the comment?